server/src/main/java/io/deephaven/server/session/SessionState.java

```diff
@@ -761,7 +761,11 @@ private synchronized void setState(final ExportNotification.State state) {
                 if (errorId == null) {
                     assignErrorId();
                 }
-                safelyExecute(() -> errorHandler.onError(state, errorId, caughtException, dependentHandle));
+                try {
+                    errorHandler.onError(state, errorId, caughtException, dependentHandle);
+                } catch (final Exception err) {
+                    log.debug().append("Unexpected error while reporting state failure: ").append(err).endl();
```

Comment:
Oops, I think we want this log to shout as `log.error()` -- since it implies that the error-handler itself is broken.